resources/css/navigation.css

```diff
@@ -1,9 +1,3 @@
-@tailwind base;
- 
-@tailwind components;
- 
-@tailwind utilities;
-
 .header{
 	
 }
```

Comment:
Remove this because it isn't being used yet - we can add it back in later if we need it

Reply:
removed